src/screens/Moderation/index.tsx

```diff
@@ -199,6 +200,21 @@ export function ModerationScreenInner({
           a.overflow_hidden,
           t.atoms.bg_contrast_25,
         ]}>
+        <Link
+          label={_(msg`View your default post interaction settings`)}
+          testID="interactionSettingsBtn"
+          to="/moderation/interaction-settings">
+          {state => (
+            <SubItem
+              title={_(msg`Interaction Settings`)}
```

Comment:
```suggestion
              title={_(msg`Interaction settings`)}
```
Hopefully I'm not being rude by giving feedback on a WIP PR, I had a look on the preview branch and I think it would fit in better with the other options with a lowercase `s`:

![IMG_9475](https://github.com/user-attachments/assets/d7b4cca2-a580-4d57-9099-ed80649b7bf4)